### PR TITLE
Polish localization

### PR DESCRIPTION
**Pull request by TacoConKvass**

### CODE REVIEW

**Comment by direwolf420** [comment]
Hey, I appreciate you doing some of my mod's translations. Did you see the "bounty" I have put up [on discord](https://discord.com/channels/103110554649894912/1141413333661138954/1141413333661138954)? The latter has some information about rewards. Contact me there please.

**Comment by TacoConKvass** [comment]
No need, I just wanted to help, thanks for repeating the offer tho.

File: Localization/pl-PL.hjson [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-			// WingVertical.DisplayName: Vertical Acceleration
-			// Class.DisplayName: Same Class
-			// Regeneration.DisplayName: Life Regeneration
+		        Damage.DisplayName: Obrażenia
+			CritChance.DisplayName: Szansa na cios krytyczny
+			Speed.DisplayName: Szybkość użycia
+			Knockback.DisplayName: Odrzut
+			FishingPower.DisplayName: Moc wędki
+			BaitPower.DisplayName: Moc przynęty
+			Defense.DisplayName: Ochrona
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: Typo, Obrona

File: Localization/pl-PL.hjson [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+			PickaxePower.DisplayName: Moc kilofa
+			AxePower.DisplayName: Moc siekiery
+			HammerPower.DisplayName: Moc młota
+			HealLife.DisplayName: Odzyskiwane życie
+			HealMana.DisplayName: Odzyskiwana mana
+			UseMana.DisplayName: Użycie many
+			Autoswing.DisplayName: Autoużycie
+			CanFishInLava.DisplayName: Umożluwia łowienie w lawie
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: Typo, Umożliwia

File: Localization/pl-PL.hjson [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
 		Configs: {
 			CISClientConfig: {
-				// DisplayName: Client Config
+				DisplayName: Konfiguracja użytkownika
 
 				AlwaysShowComparison: {
-					// Label: Always Show Comparison
-					// Tooltip: If on, always displays comparison if it exists. Use the 'Auto Select' key to toggle it otherwise (usually <LeftShift>)
+					Label: Zawsze pokazuj porównanie
+                                        Tooltip: Jeżeli włączone, porównanie będzie zawsze poakzywane, pod warunkiem, że jest możliwe do przeprowadzenia. Uźyj przycisku 'Automatyczny wybór', aby przełączyć (domyślnie <LeftShift>).
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: Typo, pokazywane

File: Localization/pl-PL.hjson (hunk 1 above)
Comment (TacoConKvass) [reply Localization/pl-PL.hjson]: Interchangeable in Polish, but will change

**Comment by TacoConKvass** [comment]
Fixed

**Comment by TacoConKvass** [comment]
Fixed v.2

File: Localization/pl-PL.hjson (hunk 1 above)
Comment (direwolf420) [reply Localization/pl-PL.hjson]: Obrona/Obrony is what is used in tooltips

File: Localization/pl-PL.hjson [hunk 4]
```diff
@@ -1,56 +1,57 @@
...
+			BaitPower.DisplayName: Moc przynęty
+			Defense.DisplayName: Obrona
+			PickaxePower.DisplayName: Moc kilofa
+			AxePower.DisplayName: Moc siekiery
+			HammerPower.DisplayName: Moc młota
+			HealLife.DisplayName: Odzyskiwane życie
+			HealMana.DisplayName: Odzyskiwana mana
+			UseMana.DisplayName: Użycie many
+			Autoswing.DisplayName: Autoużycie
+			CanCatchLavaCritters.DisplayName: Umożliwia łapanie ognistych stworzeń
```
Comment (direwolf420) [inline Localization/pl-PL.hjson]: Should be `Można w nią złapać stworzenia lawy` as per vanilla translations file (golden bug net)

Comment (TacoConKvass) [reply Localization/pl-PL.hjson]: Done

File: Localization/pl-PL.hjson [hunk 5]
```diff
@@ -1,56 +1,57 @@
...
 		Configs: {
 			CISClientConfig: {
-				// DisplayName: Client Config
+				DisplayName: Konfiguracja użytkownika
 
 				AlwaysShowComparison: {
-					// Label: Always Show Comparison
-					// Tooltip: If on, always displays comparison if it exists. Use the 'Auto Select' key to toggle it otherwise (usually <LeftShift>)
+					Label: Zawsze pokazuj porównanie
+                                        Tooltip: Jeżeli włączone, porównanie będzie zawsze pokazywane, pod warunkiem, że jest możliwe do przeprowadzenia. Uźyj przycisku 'Automatyczny wybór', aby przełączyć (domyślnie <LeftShift>).
```
Comment (direwolf420) [inline Localization/pl-PL.hjson:47]: missed this whitespace change, but merged already, no big deal

**Pull request merged**